### DISCUSSION TITLE
Featured photo

### DISCUSSION THREAD

**Discussion by mollyhormann** [CollectionBuilder/collectionbuilder.github.io#123] [General]
My featured photo will not show up no matter what I change it to. Are my photos too big?

https://mollyhormann.github.io/London-metadata/

**Comment by evanwill** [comment]
@mollyhormann it seems like you have it figured out??

**Comment by mollyhormann** [comment]
Yes, so sorry for the mix up! Thank you for all your help!

From: EvanWill ***@***.***>
Date: Tuesday, April 23, 2024 at 4:04 PM
To: CollectionBuilder/collectionbuilder.github.io ***@***.***>
Cc: Hormann, Molly M ***@***.***>, Mention ***@***.***>
Subject: [External] Re: [CollectionBuilder/collectionbuilder.github.io] Featured photo (Discussion #123)

@mollyhormann<https://github.com/mollyhormann> it seems like you have it figured out??
